uctMaths/competition/views.py

```diff
@@ -33,7 +34,7 @@ def printer_entry_result(request, school_list=None):
     #had to easy_install html5lib pisa
     #Create the HttpResponse object with the appropriate PDF headers.
     temp_school_list = []
-    if not school_list: #If not called by the admin
+    if not school_list and not request == None: #If not called by the admin
```

Comment:
Who else is calling this function that you need to add `not request == None` here?

Reply:
I think I put this in when I was trying to remove the answer sheet generation entirely from compadmin to views, but I tested and it's no longer necessary. I'll remove this now